must-gather/collection-scripts/gather_nodes

```diff
@@ -113,14 +113,19 @@ wait
 
 # Collect journal logs for specified units for all nodes
 NODE_UNITS=(kubelet crio)
-for NODE in $(oc get nodes --no-headers -o custom-columns=':metadata.name'); do
+NODES=$(oc get nodes --no-headers -o custom-columns=':metadata.name')
+for NODE in $NODES; do
     NODE_PATH=${NODES_PATH}/$NODE
     mkdir -p "${NODE_PATH}"
     for UNIT in "${NODE_UNITS[@]}"; do
         oc adm node-logs "$NODE" -u "$UNIT" > "${NODE_PATH}/${NODE}_logs_$UNIT" &
     done
+    # virtiofsd is in syslog and started by shim so -u won't work
```

Comment:
PR description says this is for documentation.  It might be better to submit code changes separately.

Reply:
Yes.  I've removed all the virtiofsd from the documentation and the changes to collect it from gather-nodes.